README.md

```diff
@@ -205,6 +208,55 @@ gh api \
 jq '[.[] | {login}]'
 ```
 
+This generates the file from all users in a GitHub organization. The same
+concept could also be used for a GitHub team. Of course, the JSON file can be
+generated by any means, as long as it follow the expected format.
+
+##### Inputs
+
+One of `team`, `local-members-path`, or `members-url` must be set. If multiple
+of these inputs are set, only one will be used. Which one will be used is
+undefined, but deterministic.
+
+**team**: This is actually the org and team formateted as ${org}/${team-slug}
+e.g. abcxyz/seahawks-2013.
+
+**local-members-path**: JSON file that exists in the repository that is calling
+the workflow.
+
+**members-url**: Full URL to a members JSON file. If this is in a public GitHub
+repository, the URL should be similar to
+https://raw.githubusercontent.com/${owner}/${repo}/${ref}/path/to/members.json.
+
+##### Teams
+
+If you use the `team` input, you will have to setup
+[github-token-minter](https://github.com/abcxyz/github-token-minter)  to provide
```

Comment:
Am I right about the requirements below?

- The usage of teams here needs github-token-minter.
- github-token-minter needs Terraform (https://github.com/abcxyz/github-token-minter)?

Reply:
I also see this code though:
```yaml
          # Use the Minty token if it was generated, otherwise use the default.
          github-token: '${{ steps.mint-token.outputs.token || github.token }}'
```

Reply:
Yes, use of the `team` input will require github-token-minter. You won't need one per repo, but you will need one somewhere. @bradegler can speak more to this.